SDL2-CS.csproj.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>SDL2-CS-Rolling</id>
+    <version>1.0.0-alpha%timestamp%</version>
+    <authors>flibitijibibo</authors>
+    <owners>flibitijibibo</owners>
```

Comment:
I don’t want to take any responsibility for NuGet binaries so while the author can be me, you’ll need to take as much ownership as possible so I never ever get reports fot binary distributions.

Reply:
Should any others be in that field? Or is just you fine?

Reply:
I think just me is okay - there’s been the odd patch or two over the years but not quite at the level that FAudio is at, for example.

Reply:
Oh wait, think we got mixed up - I can go in as the author and you can go in as the owner. Does that make sense...? I dunno how that’ll look on their site/database.

Reply:
I've made the change so that I'm the only name in the owner field.